server/src/Q3Server/ObjectGetterCached.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Runtime.Caching;
+using Q3Server.Interfaces;
+
+namespace Q3Server
+{
+    public class CachedObjectGetter<T> : IObjectGetter<T>
+    {
+        private ObjectCache Cache { get; set; }
+        private IObjectGetter<T> ObjectGetter { get; set; }
+        private double ExpirationInHours { get; set; }
```

Comment:
This is OK, but in general I think people overdo cache lifetimes so if it was me I'd always make it configurable down to minutes / seconds. If nothing else it aids testing, and the incremental value of longer cache lifetimes in production is small once you go beyond a certain point (typically a few minutes)